saleor/graphql/account/mutations/base.py

```diff
@@ -39,7 +41,29 @@ def can_edit_address(user, address):
     )
 
 
-class SetPassword(CreateToken):
+class AccountErrorMixin:
+    account_errors = graphene.List(
+        graphene.NonNull(AccountError),
+        description="List of errors that occurred executing the mutation.",
+    )
+
+    @classmethod
+    def handle_typed_errors(cls, errors: list, **extra):
```

Comment:
For every mixing like this, we need to override the entire method and the implementation is the same in every case. What if we kept this implementation in the BaseMutation only and in the mixins just provided a constant like `ERROR_TYPE_CLASS=AccountError` etc that we could use in `BaseMutation`? I would experiment with something like this to avoid code duplication.

Reply:
I wanted to do it that way, but I've stuck with that at some point. I'll refactor it.

Reply:
I've added the second field to the `cls`, for typed error field name. I think it is more readable than e..g searching class attribute with some special name.